doc/.nvmrc

```diff
@@ -0,0 +1 @@
+echo "18" > doc/.nvmrc
```

Comment:
The version itself should be in the `.nvmrc`

Reply:
Done